pkg/reporting/revision_data.go

```diff
@@ -28,6 +28,7 @@ func CommitMessageSummary(message string) string {
 		summary = fmt.Sprintf("%s...", summary[:47])
 	}
 	// escape double quotes
+	summary = strings.Replace(summary, `\`, `\\`, -1)
```

Comment:
You can not just replace every backslash maybe it already is escaping something diffenent. I think both Replace calls should be omitted and instead 

https://gohugo.io/functions/htmlescape/

should be used.

Reply:
Hmm but the htmlEscape doesn't escape the backslash